File: Module.Frontend.TPM/Client/model/tpm/commercialnet/HistoricalCommercialNet.js
Ext.define('App.model.tpm.commercialnet.HistoricalCommercialNet', {
    extend: 'Ext.data.Model',
    idProperty: '_Id',
    breezeEntityType: 'CommercialNet',
    fields: [
        { name: '_Id', type: 'string', hidden: true },
        { name: '_ObjectId', hidden: true },
        { name: '_User', type: 'string', isDefault: true },
        { name: '_Role', type: 'string', isDefault: true },
        { name: '_EditDate', type: 'date', isDefault: true },
        { name: '_Operation', type: 'string', isDefault: true },
        { name: 'Name', type: 'string', hidden: false, isDefault: true }
    ],
    proxy: {
        type: 'breeze',
        resourceName: 'HistoricalCommercialNets',
        reader: {
            type: 'json',
            totalProperty: 'inlineCount',
            root: 'results'
        }
    }
});

File: Module.Persist.TPM/Model/History/HistoricalCommercialNet.cs
using Core.History;
using Module.Persist.TPM.Model.TPM;

namespace Module.Persist.TPM.Model.History
{
    [AssociatedWith(typeof(CommercialNet))]
    public class HistoricalCommercialNet : BaseHistoricalEntity<System.Guid>
    {
        public string Name { get; set; }
    }
}
